v6-glm-py/partial_betas.py

```diff
@@ -0,0 +1,148 @@
+"""
+This file contains all partial algorithm functions, that are normally executed
```

Comment:
```suggestion
This file contains partial algorithm functions, that are normally executed
```

Reply:
Also one partial file has this header, the other not `v6-glm-py/partial_deviance.py`

Reply:
I just deleted this header as it's not so informative. It was auto-generated from the algorithm template